tests/observability/metrics/utils.py

```diff
@@ -650,8 +650,10 @@ def get_vm_cpu_info_from_prometheus(prometheus: Prometheus, vm_name: str) -> Opt
     return None
 
 
-def validate_vmi_node_cpu_affinity_with_prometheus(prometheus: Prometheus, vm: VirtualMachineForTests) -> None:
-    vm_cpu = vm.instance.spec.template.spec.domain.cpu
+def validate_vmi_node_cpu_affinity_with_prometheus(
+    prometheus: Prometheus, vm: VirtualMachineForTests, windows=False
+) -> None:
+    vm_cpu = vm.vmi.instance.spec.domain.cpu if windows else vm.instance.spec.template.spec.domain.cpu
```

Comment:
not clear for me - why we have difference for rhel and windows here?
I suppose we can use `vm.vmi.instance.spec.domain.cpu` in both cases